Jenkinsfile

```diff
@@ -13,8 +13,12 @@ def k8s_job=""
 xray_projectkey='MQ'
 xray_on_demand_testplan='MQ-1'
 xray_nightly_testplan='MQ-17'
+xray_continuous_testplan='MQ-33'
 xray_test_execution_type='10059'
 
+// if e2e run does not build its own images, which tag to use when pulling
```

Comment:
rename to e2e_continuous_image_tag, then usage is clearer

Reply:
Now fixed